keras/models.py

```diff
@@ -76,6 +88,49 @@ def standardize_weights(y, sample_weight=None, class_weight=None):
     else:
         return np.ones(y.shape[:-1] + (1,))
 
+def sequential_from_yaml(yamlString):
+    '''
+        Returns a compiled Sequential model generated from a local yaml file,
+        which is either created by hand or from Sequential.to_yaml
+    '''
+    modelYaml = yaml.load(yamlString)
+    model = Sequential()
+
+    class_mode = modelYaml.get('class_mode')
+    theano_mode = modelYaml.get('theano_mode')
+    loss = globals()[modelYaml.get('loss')]
```

Comment:
It's the same hack, but the big difference is `get` is only used in small files where the set of names you want to be able to "get" is identical (or close enough) to the entire namespace. Which makes it safe. The reason this hack is potentially dangerous in the first place is the likelihood of namespace collisions, and they are highly likely in a file as busy as `models.py`. 

So I'd rather import each module separately, then used `getattr` on the appropriate module name. Then we limit the use of the namespace to the name of modules, which is fine.